pkg/virt-handler/hotplug-disk/findmnt.go

```diff
@@ -80,14 +81,22 @@ func parseMntInfoJson(mntInfoJson []byte) ([]FindmntInfo, error) {
 	return mntinfo.Filesystems, nil
 }
 
-func (f *FindmntInfo) GetSource() string {
+func (f *FindmntInfo) GetSourcePath() string {
 	match := sourceRgx.FindStringSubmatch(f.Source)
 	if len(match) != 2 {
 		return strings.TrimPrefix(f.Source, rhcosPrefix)
 	}
 	return strings.TrimPrefix(match[1], rhcosPrefix)
 }
 
+func (f *FindmntInfo) GetSourceDevice() string {
+	match := deviceRgx.FindStringSubmatch(f.Source)
+	if len(match) == 0 {
```

Comment:
Nit: we really want `len(match)` to be exactly 2, like in the function above.

Reply:
I guess I can flip the logic to
```go
if len(match) == 2 {
  return match[1]
}
return ""
```